test/post_fetch_helpers.jl

```diff
@@ -0,0 +1,35 @@
+using Test
+using DataDeps: unpack, p7zip
+
+# Runs `f()` and returns what would normally be printed to stdout
+function capture_stdout(f)
+    std = stdout
+    r, w = redirect_stdout()
+    try
+        f()
+    finally
+        redirect_stdout(std)
+        close(w)
+    end
+    output = read(r, String)
+    return output
```

Comment:
Can't we do: `const capture_stdout = sprint ∘ redirect_stdout` ?

Reply:
interesting, let me look into that (copy pasting this results in an error, but may be workable)

Reply:
I still think there must be a much shorter way to do this.
Did you get a chance to look at it?